### DISCUSSION TITLE
SOAP WS - Is supported?

### DISCUSSION THREAD

**Discussion by EsterDaci** [micronaut-projects/micronaut-core#11247] [Q&A]
Hello,

I am about to implement a soap web service in Micronaut, using Kotlin.
So far I have searched, but have not found a useful documentation, about how to implement this in Micronaut.  Is it any suggestions, if anyone has done before? What libraries have you used? I would really appreciate any help, because our client only support SOAP.

**Comment by nedelva** [comment]
The question has been asked already. AFAIK the answer is a resounding **NO**.
That being said, you might find [this older discussion](https://github.com/micronaut-projects/micronaut-core/discussions/9662) useful.
